lib/pf/Switch/OpenVPN.pm

```diff
@@ -0,0 +1,163 @@
+package pf::Switch::OpenVPN;
```

Comment:
#   Failed test 'no tab character in /usr/local/pf/lib/pf/Switch/OpenVPN.pm'
#   at coding-style.t line 49.
# Looks like you failed 1 test of 1051.
coding-style.t .....................................................